css/helpers.css

```diff
@@ -0,0 +1,112 @@
+/* TODO(v12) remove some of these in favor of v2 styling */
```

Comment:
Foundry v12 has more and better utility classes. Sadly v11 does not so I made the ones I needed instead.